docs/datatips.md

```diff
@@ -20,7 +20,7 @@ Consume the datatip [Atom service](http://flight-manual.atom.io/behind-atom/sect
 "consumedServices": {
   "datatip": {
     "versions": {
-      "0.1.0": "consumeDatatipService"
+      "0.1.0": "consumeDatatip"
```

Comment:
See line 32 below - the two have to match. If you're going to change this doc both of these have to change.

Reply:
Done in 7b8e4c1